Code/Tools/ProjectManager/Source/GemCatalog/GemListView.cpp

```diff
@@ -31,6 +34,8 @@ namespace O3DE::ProjectManager
                         this->viewport()->repaint();
                     });
             });
+        
+        connect(header, &AdjustableHeaderWidget::sectionsResized, [=] { repaint(); });
```

Comment:
See the comment above, I think you could also just do an `updateGeometry()` above instead of having this logic to force a repaint.

Also of note, if you do need to stick with this logic, it's preferable to call `update()` instead of `repaint()` for performance reasons/to avoid flickering:  https://doc.qt.io/qt-5/qwidget.html#update

Reply:
I switched to using update() and it looks like there is no flickering with that either so will keep that. Yea I also ran into issues where I needed to delay the resize of sections with a singleShot so I am for keep this logic. But again changed to update still works here and in the GemListView.

Reply:
Great thanks! See my response about the resizing above, but in brevity I am OK with it.